feat: initial write-up of Budgie 10.10 post (#10)
BuddiesOfBudgie/site: src/content/blog/budgie-10-10-released.mdx

- **Desktop Backgrounds**: Handled by [swaybg](https://github.com/swaywm/swaybg) to ensure wallpapers display correctly under Wayland.
- **Application Integration**: Uses XDG Desktop Portals [xdg-desktop-portal-gtk](https://github.com/flatpak/xdg-desktop-portal-gtk) for most tasks and [xdg-desktop-portal-wlr](https://github.com/emersion/xdg-desktop-portal-wlr) for screenshots and ScreenCasts. This enables applications to request screen sharing in a secure, standardized way.

For display configuration, we recommend pairing Budgie 10.10 with [wdisplays](https://github.com/artizirk/wdisplays) to manage multiple monitors and resolutions. Our new [Budgie Desktop Services](https://forge.moderndesktop.dev/BuddiesOfBudgie/budgie-desktop-services) (written in Qt6 and C++) automatically saves your settings so they persist across sessions. This component will serve as the "beating heart" of Budgie 11; for Budgie 10.10, its primary focus is enabling persistent display configuration.

> #### Compositor Recommendation and Bridge

Reviewer: I'm not sure that this needs to be a subsection of "Leveraging the Wayland Ecosystem".

```suggestion
### Compositor Recommendation and Bridge
```

For Budgie 10.10, we recommend the use of a wlroots-based compositor. The team has put special effort into enabling a great experience with [labwc](https://labwc.github.io/), a modern, lightweight, and feature-rich Wayland compositor perfectly suited for Budgie Desktop.

We have achieved this by implementing a "labwc bridge" that automatically configures labwc with (in our opinion) sensible defaults. It mirrors settings from Budgie Control Center and Budgie Desktop Settings to labwc-specific configuration files in `~/.config/budgie-desktop/labwc` (while also supporting `XDG_CONFIG_HOME`). This bridge enables:

- **Default keyboard shortcuts** for essential actions like opening the Budgie Menu, Raven, the Budgie Run Dialog, workspace switching, and screenshotting.
- **Input acceleration** for both touchpads and mice.
- **Consistent theming** for fonts, drop shadows, and title bar layouts.
- **Specific window rules** for the desktop, power dialog, and budgie-daemon.
- **Window management features** like snapping and a window switcher with thumbnails.

> Beyond our recommendation default of `labwc`, the move to Wayland represents a fundamental architectural shift. This shift to a protocol-first architecture represents a major milestone: it decouples the desktop from a specific window manager (budgie-wm, which in turn relied on our Magpie fork of Mutter) and makes Budgie truly compositor-agnostic, opening the door for experimentation with alternative compositors beyond our primary recommendations.

Reviewer: ```suggestion
Beyond our recommended default of `labwc`, the move to Wayland represents a fundamental architectural shift. This shift to a protocol-first architecture represents a major milestone: it decouples the desktop from a specific window manager (`budgie-wm`, which in turn relied on our fork of Mutter called Magpie), and makes Budgie truly compositor-agnostic, opening the door for experimentation with alternative compositors beyond our primary recommendations.
```

- **Notifications**: You can now toggle Do Not Disturb mode by middle-clicking the Notifications applet icon.
- **Tasklist**: Our more "legacy" style Tasklist saw a **complete rewrite**, moving away from merely being a `Wnck` Tasklist to a more modern implementation that is built on top of libxfce4windowing. The new Tasklist scales better with many applications, including dynamic button sizing and dynamic scrolling (left and right navigation buttons also appear for more ease-of-use).
- **Workspaces**: Improved reliability when new windows are opened and made the add workspace button more reactive.

### Panel Updates and Improvements

`budgie-panel` now leverages `layer-shell` to anchor itself to the screen edges. We also improved the logic for start, center, and end sections to dynamically calculate heights / widths (depending on orientation), ensuring inner applets like the legacy Tasklist trigger overflow behavior correctly.

### Other Component Updates

Our `budgie-desktop` was not the only component that received updates to enable Wayland support!

#### Budgie Desktop View

> Budgie Desktop View supports Wayland, leveraging `layer-shell` to ensure it remains positioned behind windows, Raven, and the panel.

Reviewer: ```suggestion
Budgie Desktop View now supports Wayland, leveraging `layer-shell` to ensure it remains positioned behind windows, Raven, and the panel.
```
